webpack/scenes/ContentViews/Publish/CVPublishFinish.js

```diff
@@ -161,7 +163,8 @@ const CVPublishFinish = ({
   }
   if (status === STATUS.PENDING) return (<Loading />);
   if (status === STATUS.ERROR) return (<EmptyStateMessage error={error} />);
-  return <Loading />;
+  const { input: { content_view_version_id: cvvID } } = response;
+  return <Redirect to={`/content_views/${cvId}#/versions/${cvvID}`} />;
```

Comment:
This doesn't handle the case where the wizard is closed manually. It lands on the same tab and doesn't refresh the view.

Reply:
If they close tab manually, I would expect the user should not be redirect.. In my tests it does refresh upon closing it manually.